experimenter/experimenter/features/manifests/nimbus_fml_loader.py

```diff
@@ -67,6 +85,35 @@ def get_minor_version_ref(self, version):
             minor=version.minor,
             patch=version.micro,
         )
+    
+    def get_fml_clients(self, versions: list[str]) -> list[FmlClient]:
+        refs = self.get_version_refs(versions)
+        file_path = self.get_local_file_path()
+        return [self.create_client(file_path, self.channel) for r in refs]
+
+    def create_client(self, path: str, channel: str) -> FmlClient:
+        return FmlClient(path, channel)
+
+    def get_inspector(self, client: FmlClient, feature_id: str) -> FmlFeatureInspector:
+        return client.get_feature_inspector(feature_id)
+
+    def get_errors(self, inspector: FmlFeatureInspector, blob: str):
+        return inspector.get_errors(blob)
+
+    # Todo: Add versioning https://mozilla-hub.atlassian.net/browse/EXP-3875
```

Comment:
This should reference the GH issue.

Reply:
This is an epic with no issues (yet), so there is no GH issue